docs/src/api/class-locator.md

```diff
@@ -580,6 +580,19 @@ When all steps combined have not finished during the specified [`option: timeout
 ### option: Locator.dblclick.trial = %%-input-trial-with-modifiers-%%
 * since: v1.14
 
+## method: Locator.describe
```

Comment:
setDescription? Otherwise it sounds like the method is used to return a description of this locator.

Reply:
We need to keep it immutable and chained.